hal/sleep_api.h

```diff
@@ -29,13 +29,15 @@ extern "C" {
 
 /** Send the microcontroller to sleep
  *
- * The processor is setup ready for sleep, and sent to sleep using __WFI(). In this mode, the
+ * The processor is setup ready for sleep, and sent to sleep. In this mode, the
  * system clock to the core is stopped until a reset or an interrupt occurs. This eliminates
  * dynamic power used by the processor, memory systems and buses. The processor, peripheral and
  * memory state are maintained, and the peripherals continue to work and can generate interrupts.
  *
  * The processor can be woken up by any internal peripheral interrupt or external pin interrupt.
  *
+ * The wake-up time should be less than 10 us.
+ * 
  * @note
```

Comment:
I would like to remove this note as it's specific to one target, and should be documented elsewhere. Where shall we move it (or better question - where we would expect to have documented sleep modes for a target - is it all in the code, that this note would be moved to lpc targets that have localfilesystem, same for what modes, or developer target page) ?